internal/generator/vector/output/elasticsearch/elasticsearch.go

```diff
@@ -0,0 +1,133 @@
+package elasticsearch
+
+import (
+	"strings"
+
+	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+	. "github.com/openshift/cluster-logging-operator/internal/generator"
+	"github.com/openshift/cluster-logging-operator/internal/generator/fluentd/helpers"
+	vectorhelpers "github.com/openshift/cluster-logging-operator/internal/generator/vector/helpers"
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/output/security"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type Elasticsearch struct {
+	Desc           string
+	ComponentID    string
+	Inputs         string
+	Index          string
+	Endpoint       string
+	RetryTag       Element
```

Comment:
what is this used for?

Reply:
This is Elasticsearch class similar to Kafka. Remove unused fields now.